packages/tabs/src/elements/Tabs.js

```diff
@@ -8,16 +8,17 @@
 import React, { Children, cloneElement, isValidElement } from 'react';
 import PropTypes from 'prop-types';
 import { ControlledComponent } from '@zendeskgarden/react-selection';
+import { TabsContainer } from '@zendeskgarden/container-tabs';
+import { isRtl, withTheme } from '@zendeskgarden/react-theming';
 
-import TabsContainer from '../containers/TabsContainer';
 import TabsView from '../views/TabsView';
 import TabList from '../views/TabList';
 import Tab from '../views/Tab';
 
 /**
  * High-level abstraction for basic Tabs implementations.
  */
-export default class Tabs extends ControlledComponent {
+class Tabs extends ControlledComponent {
```

Comment:
Why not convert to a functional component here and use `useTabs` instead? It helps reduce code size and is easier to read. See #431

Reply:
This would be a breaking change with removing `onStateChange` and how people could have overrode it.

We can include that API change when we change the Tabs API in CSS-in-JS.